.github/workflows/actions_build.yml

```diff
@@ -18,7 +18,8 @@ concurrency:
 env:
   LC_ALL: "en_US.UTF-8"
   BUILD_JDK_VERSION: "19"
-  GRADLE_ENTERPRISE_ACCESS_KEY: ${{ secrets.GRADLE_ENTERPRISE_ACCESS_KEY }}
+  PRIVATE_GRADLE_ENTERPRISE_ACCESS_KEY: ${{ secrets.GRADLE_ENTERPRISE_ACCESS_KEY }}
+  PUBLIC_GRADLE_ENTERPRISE_ACCESS_KEY: "scr6znfepfdcwgczcbybz22tb44yae7xgjxorl32vtrlojl6x37q"
```

Comment:
Exposing secrets in this manner (or in any other way that would run an action on an external PR) would allow anyone to use the exposed secret. In this case, it would allow anyone access to your Gradle Enterprise instance, which they could (ab)use to connect their projects to and gain full benefits of it.

Reply:
Forgot to mention this 😅 
This PR is superseded by the following PR: https://github.com/line/armeria/pull/5104
(the credential has already been revoked)